life/updated/02-life.adoc

```diff
@@ -358,137 +358,144 @@ He took me by the hand, and led me by a way I knew not. He made darkness light b
+I have overcome the world.`" This has been a comfort to me many times.
+
 I often remember a saying of a faithful servant and minister of Jesus Christ,
-whose name was Miles Halhead, when I was under great exercise:
+whose name was Miles Halhead, when I was under great exercise.
 He steadfastly looking upon me, said, "`Dear child, if you continue in the truth,
 you will make an honorable woman for the Lord;
-for the Lord God will honor you with his blessed testimony.`"
+for the Lord God will honor you with His blessed testimony.`"
 And ten years after, in 1665, he came to my habitation, and said to me,
-"`My love and life is with you,
-and that for the blessed work`'s sake that is going on in you;
-the Lord God keep you faithful, for he will require harder things of you,
-than you are aware of; the Lord give you strength to perform them,
-and keep you faithful to his blessed testimony; my prayers shall be for you,
-as often as I remember you.`"
-Soon after, a great exercise fell upon us; we were exposed to much suffering,
-and the Lord had opened my mouth in a testimony but a little before.
-I have been concerned, for fear my friends should suffer for me; but not for myself;
+"`My love and life is with you, for the sake of the blessed work that is going on in you.
+May the Lord God keep you faithful, for He will require harder things of you
+than you are aware of. May He give you strength to perform them,
+and keep you faithful to His blessed testimony.
+My prayers shall be for you, as often as I remember you.`"
+
+Soon after this, the Lord opened my mouth in a testimony,
+and a great exercise fell upon us, in which we were exposed to much suffering.
+The least of what we suffered was loss of goods, beating and hurling to and fro,
+and dragging out of our meetinghouse, among many other abuses,
+which the Lord made us able to go through, and sanctified to us.
+I have been concerned for fear my friends should suffer for me, but not for myself,
 for I could truly say, "`My heart was given up to serve the Lord, come what would come.`"
-But the least of our sorrow was loss of goods, beating and hurling to and fro,
-and dragging out of our meetinghouse, and many other abuses,
-which the Lord made us able to go through, and sanctified to us;
-and my soul blesses the Lord, that he accounted us worthy to suffer for his name sake.
-
-For in the time of suffering,
-a selfish separating spirit began to break forth among us;
-which added to our affliction more than all our persecutors could do;
-though we went in great hazard of our lives to our meetings,
-the informers were so wicked and inhuman, and filled with envy and madness,
+My soul blesses the Lord, that He accounted us worthy to suffer for His name sake.
+For in this time of suffering,
+a selfish separating spirit began to break forth among us,
+which added to our affliction more than all our persecutors could do.
+We went in great hazard of our lives to our meetings,
```

Comment:
I like how you are trying to shorten these sentences. This particular one, however, seems like it loses something when you put the period and take out the word _though_. I think it might read a little more clearly if you put it like this: ".... which added to our affliction more than all our persecutors could do, **even though at this time** we went in great hazard of our lives to our meetings,...."  Because her point is that, even though things were horrible with the persecutors, the selfish spirit that broke froth among friends was even worse. The sentence remains super long... which is not the best... but at least the point remains clear.